<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>686</width>
        <height>456</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>X-Plane + ExtPlane IP &amp; port</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="serverAddressEdit">
         <property name="text">
          <string>127.0.0.1:51000</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Panel rotation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDial" name="panelRotationDial">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="minimum">
          <number>-179</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="notchTarget">
          <double>5.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox">
         <property name="minimum">
          <number>-179</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>ExtPlane update rate limit (Hz)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Panel repaint rate (FPS)</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Default font size</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Other options</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="fullscreenCheckbox">
         <property name="text">
          <string>Fullscreen</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="simulateCheckbox">
         <property name="text">
          <string>Simulated connection (requires restart)</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QCheckBox" name="interpolateCheckbox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Makes movement of needles smoother on some instruments.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Use with high update interval or disable on slow devices if needed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Interpolate values</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="antialiasCheckbox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Smooths edges of moving parts for visual clarity. Disable on slow devices.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Antialias</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="updateIntervalComboBox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How often X-Plane should send updates (at maximum). Set to as low value as possible for best preformance.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Don't change</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>60</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="panelUpdateComboBox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How often the panel should be redrawn. Set to lower value on slow devices. Has effect only if interpolation is enabled.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Update only when refs change</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>60</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="fontSizeSpinBox">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>5.000000000000000</double>
         </property>
         <property name="value">
          <double>15.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QCheckBox" name="rememberSizeAndPositionCheckBox">
         <property name="text">
          <string>Remember panel size and position</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QCheckBox" name="autoPanelsCheckbox">
         <property name="text">
          <string>Automatically create and load panels for each aircraft</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QCheckBox" name="adjustPowerCheckbox">
         <property name="text">
          <string>Adjust panel power and brightness</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panelRotationDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>panelRotationDial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
